yt/data_objects/construction_data_containers.py

```diff
@@ -180,6 +180,7 @@ def __init__(self, field, axis, weight_field=None, center=None, ds=None,
         self._set_center(center)
         self._projected_units = {}
         if data_source is None: data_source = self.ds.all_data()
+        data_source.set_field_parameter("axis", self.axis)
```

Comment:
I *think* we want this to be set in the `self` object, and assume it will get passed down during the contextmanager during the accessing of `chunks` on the `data_source`.

Reply:
Sorry, could you elaborate?

Reply:
I recall this not working if I didn't do this.

Reply:
Ok, for some reason projections of particle datasets don't work if I don't do this. Maybe the same logic for grids needs to be implemented in the particle chunking vis-a-vis field parameters?

Reply:
Never mind, fixed this